src/common/meta/src/region_keeper.rs

```diff
@@ -83,16 +83,18 @@ impl MemoryRegionKeeper {
         inner.contains(&(datanode_id, region_id))
     }
 
-    /// Returns a set of filtered out regions that are opening.
-    pub fn filter_opening_regions(
+    /// Filters out all operating regions from `region_ids` and returns operating regions.
+    pub fn filter_operating_regions(
```

Comment:
Non-blocking: How about rename it as `extract_operating_regions`? Because I found that this method seems to have done the opposite of what it originally did after being modified.